test/sagemaker_tests/pytorch/training/resources/inductor/run_pytorch_inductor.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[0])' | tr -d "'")
+if [ "$PYTHON_VERSION" -eq 2 ]
+then
+  exit 0
+fi
+HOME_DIR=/test/benchmark
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs
+
+mkdir -p ${HOME_DIR}
+mkdir -p ${BIN_DIR}
+mkdir -p ${LOG_DIR}
+
+python -c "import sys;
+assert sys.version_info < (3, 7);"
+PYTHON_VERSION_CHECK=`echo $?`
+
+set -e
+cd $BIN_DIR
+# tmp fix for numpy deprecations, PR raised: https://github.com/HewlettPackard/dlcookbook-dlbs/pull/19
+git clone --quiet --single-branch --depth 1 --branch v2.0.0 https://github.com/pytorch/pytorch.git pytorch
+
+cd pytorch
+
+pip install -U numpy
+pip install deepspeed==0.8.2
+pip install gitpython
+pip install tabulate==0.9.0
+
+TRAINING_LOG=${LOG_DIR}/pytorch_inductor_huggingface_benchmark.log
+
+python benchmarks/dynamo/runner.py --suites=huggingface --training --dtypes=amp --compilers=inductor --output-dir=huggingface_logs --extra-args='--output-directory=./' > $TRAINING_LOG 2>&1 
+
+RETURN_VAL=`echo $?`
+set -e
+
+if [ ${RETURN_VAL} -eq 0 ]; then
+    echo "Training Huggingface Complete using PyTorch Inductor."
+else
+    echo "Training Huggingface Failed using PyTorch Inductor."
+    cat $TRAINING_LOG
+    exit 1
+fi
+
+TRAINING_LOG=${LOG_DIR}/pytorch_inductor_timm_benchmark.log
+
+python benchmarks/dynamo/runner.py --suites=timm_models --training --dtypes=amp --compilers=inductor --output-dir=timm_logs --extra-args='--output-directory=./' > $TRAINING_LOG 2>&1 
```

Comment:
Set output directory to /opt/ml/output/data for the artifacts to be automatically picked up by sagemaker - https://docs.aws.amazon.com/sagemaker/latest/dg/model-train-storage.html

Upload logs to the same location.